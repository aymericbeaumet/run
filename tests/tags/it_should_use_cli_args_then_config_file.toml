tags = ["c", "b", "a"]

[[run]]
tags = ["a"]
cmd = ["bash", "-c", "echo a"]

[[run]]
tags = ["b"]
cmd = ["bash", "-c", "echo b"]

[[run]]
tags = ["c"]
cmd = ["bash", "-c", "echo c"]