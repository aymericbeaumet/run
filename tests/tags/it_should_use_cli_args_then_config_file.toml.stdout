[bash] a
[bash] b
[bash] c
